pkg/kubelet/eviction/BUILD

```diff
@@ -16,6 +16,7 @@ go_library(
         "doc.go",
         "eviction_manager.go",
         "helpers.go",
+        "threshold_notifier_unsupported.go",
```

Comment:
This block should only contain files used to build linux, amd64. We don't need this file.